src/main/resources/static/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Lavalink Config Server</title>
+    <link rel="icon" href="https://avatars.githubusercontent.com/u/133400169?s=200&v=4" type="image/png">
```

Comment:
ideally add this statically https://github.com/lavalink-devs/Lavalink/blob/master/branding/lavalink.svg
also favicon